src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd

```diff
@@ -199,6 +202,11 @@ echo.
 echo All operations completed successfully!
 goto :Done
 
+:Unlock
+call :Run %_appcmd% unlock config /section:%1
+if ERRORLEVEL 1 goto :Failure
```

Comment:
This may cause problem as technically we are still in `Unlock` function when we do this jump in case of error. I would recommend we stick with calling `:Run` function directly. We will have to repeat `if ERRORLEVEL ...` a few times, but we avoid potential weirdness/headache when things go wrong.